Dockerfile.slim

```diff
@@ -1,9 +1,8 @@
 FROM alpine:3.12
 LABEL maintainer="sig-platform@spinnaker.io"
 
-ENV KUSTOMIZE_VERSION=3.8.6
```

Comment:
Removing this breaks [this I'd think.  ](https://github.com/spinnaker/rosco/blob/2c370634f065928fdabfcf2396e7031d74d2a1d6/Dockerfile.slim#L34)